.github/workflows/pip-python-tests.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package test
```

Comment:
I don't think we need a separate pip vs python tests file?  The conda test file isn't installing the package from conda but pip.  It's just using conda environments instead of python to set everything up.  The reason for using conda is that a lot of libraries, like pytorch, run much faster when installed from conda - so our tests will run much faster.

Reply:
Totally agree on that conda is much faster. But we are validating using conda packages but then publishing a pypi package which I think are different scenarios given that dependencies in pip and conda might be different.